static/account_explanation.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Impact Account</title>
+
+    <!-- CSS  -->
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"/>
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" type="text/css" href="/min/style-min.css"/>
+</head>
+<body>
+<header>
+    <div class="navbar-fixed">
+        <nav role="navigation">
+            <div class="nav-wrapper container">
+                <a href="/" class="brand-logo"><h1>Impact</h1></a>
+            </div>
+        </nav>
+    </div>
+</header>
+
+<section class="section container">
+    <div class="row">
+        <h4 class="col s12">How Impact Accounts work</h4>
+        <div class="col s12">
+            <h5>The current system</h5>
+            <p>
+                The newer (4.8+) Impact Account system separates enabling premium and listing info (such as capes). It also adds a layer of privacy to showing info.
+            </p><p>
+                <a href="//api.impactclient.net/v1/minecraft/user/info?pretty=1" target="_blank">Our API shows a SHA hashed version of your Minecraft ID</a>, which prevents somebody from easily getting a list of Impact Account holders.
+                However, it is still possible for them to check if individual Minecraft accounts have Impact Account features.
+                They can do this by hashing the Minecraft ID and then looking through the API's list to see if the resultant hash is in the list.
+                <b>If you don't want this to be possible, you can enable Incognito Mode</b> and you won't be included in the list.
+                Not being in the list means no cape will be shown and noone can tell if you have an Impact Account or not.
```

Comment:
maybe make it `no one can tell if you have an Impact Account or not, not even other Impact users.`